actionview/lib/action_view/template.rb

```diff
@@ -375,11 +376,11 @@ def strict_locals?
     # to ensure that references to the template object can be marshalled as well. This means forgoing
     # the marshalling of the compiler mutex and instantiating that again on unmarshalling.
     def marshal_dump # :nodoc:
-      [ @source, @identifier, @handler, @compiled, @locals, @virtual_path, @format, @variant ]
+      [ @_source, @identifier, @handler, @compiled, @locals, @virtual_path, @format, @variant ]
```

Comment:
My fault for not mentioning these in the review, they'll need to be changed back as well
```suggestion
      [ @source, @identifier, @handler, @compiled, @locals, @virtual_path, @format, @variant ]
```

Reply:
Fixed, I should have caught that myself